charts/quickwit/values.yaml

```diff
@@ -423,7 +499,7 @@ seed:
     #     params:
     #       topic: quickwit-topic
     #       client_params:
-    #         bootstrap.servers: kafka-server-endpoint1:9092,kafka-server-endpoint2:9092
+  #         bootstrap.servers: kafka-server-endpoint1:9092,kafka-server-endpoint2:9092
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed